src/components/Layout/__snapshots__/Layout.test.tsx.snap

```diff
@@ -246,16 +235,55 @@ exports[`rendering of Layout Component > checks if component renders 1`] = `
               <li
                 className="usa-nav__primary-item"
               >
-                <a
-                  aria-current="page"
-                  className="usa-current"
-                  href="/"
+                <button
+                  aria-controls="scorecardMenu"
+                  aria-expanded={false}
+                  className="usa-accordion__button usa-nav__link usa-current"
+                  data-cy="nav-dropdown-scorecard-nav"
```

Comment:
what is data-cy? I've seen it and I'm just like "who are you"? Is it for testing?

Reply:
yes, data-cy allows for easy DOM node selection while testing with cypress. Hoping to get cypress testing in for integration tests after we stabilize a bit.